README.md

```diff
@@ -684,6 +684,203 @@ var callbacks = {
 BillForward.addPayVisionForm(formSelector, supportedCardBrands, wpwlOptions, callbacks);
 ```
 
+### Capture Bank Account
+
+For the moment the only bank account capture supported is in `stripe` using `ACH`.
+When capturing a bank account the flow can be divided in two steps:
+
+- Capturing
+- Verification
+
+Once the `Capture` has been made, `stripe` will deposit two random amounts on the bank account. These amounts have to be
+verified against (or in other words, *sent to*) `stripe`. You can do the verification process directly using `billforward-js`.
```

Comment:
Inline code formatting makes `stripe` seem like an enum constant.

Prefer:
Stripe
here; we are talking about the Stripe servers/API/business — it is a proper noun, not a code concept.